### DISCUSSION TITLE
novice in python/programming, want to click buttons on websites

### DISCUSSION THREAD

**Discussion by pixel-87** [seleniumbase/SeleniumBase#2260] [Q&A] [answered]
Hi, sorry if this isn't the usual format.
I want to make a bot that automatically clicks a couple button on a website daily and this seems like a good repo not to get caught by bot detectors, but I'm not experienced enough to read and understand what is written here and how to use it so does anyone have any pointers on what to do.
I want the bot to at a semi-similar time each day (not exactly the same time to avoid suspicion) click a few buttons and then close the page and open another and repeat these steps a couple of times.
thanks for any help!

**Comment by mdmintz** [comment, marked as answer]
Hello, I'd start with a few example tests that click buttons undetected:
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/raw_turnstile.py
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/raw_form_turnstile.py

Be sure you're using the latest version of SeleniumBase (currently `4.21.1`)

**Reply by pixel-87** [reply]
Thanks :)
